internal/driver/mobile/event/key/code_string.go

```diff
@@ -1,60 +1,183 @@
+	_ = x[CodeF21-112]
+	_ = x[CodeF22-113]
+	_ = x[CodeF23-114]
+	_ = x[CodeF24-115]
+	_ = x[CodeHelp-117]
+	_ = x[CodeMute-127]
+	_ = x[CodeVolumeUp-128]
+	_ = x[CodeVolumeDown-129]
+	_ = x[CodeLeftControl-224]
+	_ = x[CodeLeftShift-225]
+	_ = x[CodeLeftAlt-226]
+	_ = x[CodeLeftGUI-227]
+	_ = x[CodeRightControl-228]
+	_ = x[CodeRightShift-229]
+	_ = x[CodeRightAlt-230]
+	_ = x[CodeRightGUI-231]
+	_ = x[CodeCompose-65536]
+}
 
 const (
-	codeName0 = "CodeUnknown"
-	codeName1 = "CodeACodeBCodeCCodeDCodeECodeFCodeGCodeHCodeICodeJCodeKCodeLCodeMCodeNCodeOCodePCodeQCodeRCodeSCodeTCodeUCodeVCodeWCodeXCodeYCodeZCode1Code2Code3Code4Code5Code6Code7Code8Code9Code0CodeReturnEnterCodeEscapeCodeDeleteBackspaceCodeTabCodeSpacebarCodeHyphenMinusCodeEqualSignCodeLeftSquareBracketCodeRightSquareBracketCodeBackslash"
-	codeName2 = "CodeSemicolonCodeApostropheCodeGraveAccentCodeCommaCodeFullStopCodeSlashCodeCapsLockCodeF1CodeF2CodeF3CodeF4CodeF5CodeF6CodeF7CodeF8CodeF9CodeF10CodeF11CodeF12"
-	codeName3 = "CodePauseCodeInsertCodeHomeCodePageUpCodeDeleteForwardCodeEndCodePageDownCodeRightArrowCodeLeftArrowCodeDownArrowCodeUpArrowCodeKeypadNumLockCodeKeypadSlashCodeKeypadAsteriskCodeKeypadHyphenMinusCodeKeypadPlusSignCodeKeypadEnterCodeKeypad1CodeKeypad2CodeKeypad3CodeKeypad4CodeKeypad5CodeKeypad6CodeKeypad7CodeKeypad8CodeKeypad9CodeKeypad0CodeKeypadFullStop"
-	codeName4 = "CodeKeypadEqualSignCodeF13CodeF14CodeF15CodeF16CodeF17CodeF18CodeF19CodeF20CodeF21CodeF22CodeF23CodeF24"
-	codeName5 = "CodeHelp"
-	codeName6 = "CodeMuteCodeVolumeUpCodeVolumeDown"
-	codeName7 = "CodeLeftControlCodeLeftShiftCodeLeftAltCodeLeftGUICodeRightControlCodeRightShiftCodeRightAltCodeRightGUI"
-	codeName8 = "CodeCompose"
+	_Code_name_0 = "CodeUnknown"
+	_Code_name_1 = "CodeACodeBCodeCCodeDCodeECodeFCodeGCodeHCodeICodeJCodeKCodeLCodeMCodeNCodeOCodePCodeQCodeRCodeSCodeTCodeUCodeVCodeWCodeXCodeYCodeZCode1Code2Code3Code4Code5Code6Code7Code8Code9Code0CodeReturnEnterCodeEscapeCodeDeleteBackspaceCodeTabCodeSpacebarCodeHyphenMinusCodeEqualSignCodeLeftSquareBracketCodeRightSquareBracketCodeBackslash"
+	_Code_name_2 = "CodeSemicolonCodeApostropheCodeGraveAccentCodeCommaCodeFullStopCodeSlashCodeCapsLockCodeF1CodeF2CodeF3CodeF4CodeF5CodeF6CodeF7CodeF8CodeF9CodeF10CodeF11CodeF12"
+	_Code_name_3 = "CodePauseCodeInsertCodeHomeCodePageUpCodeDeleteForwardCodeEndCodePageDownCodeRightArrowCodeLeftArrowCodeDownArrowCodeUpArrowCodeKeypadNumLockCodeKeypadSlashCodeKeypadAsteriskCodeKeypadHyphenMinusCodeKeypadPlusSignCodeKeypadEnterCodeKeypad1CodeKeypad2CodeKeypad3CodeKeypad4CodeKeypad5CodeKeypad6CodeKeypad7CodeKeypad8CodeKeypad9CodeKeypad0CodeKeypadFullStop"
+	_Code_name_4 = "CodeKeypadEqualSignCodeF13CodeF14CodeF15CodeF16CodeF17CodeF18CodeF19CodeF20CodeF21CodeF22CodeF23CodeF24"
+	_Code_name_5 = "CodeHelp"
+	_Code_name_6 = "CodeMuteCodeVolumeUpCodeVolumeDown"
+	_Code_name_7 = "CodeLeftControlCodeLeftShiftCodeLeftAltCodeLeftGUICodeRightControlCodeRightShiftCodeRightAltCodeRightGUI"
+	_Code_name_8 = "CodeCompose"
```

Comment:
FYI: While I merge gomobile in, these names were rewritten to meet the CI lint requirements. Currently, it seems the requirement was removed, and these naming style is acceptable :)

Reply:
Same in other places.